src/substrate-indexed-data/orders/order.service.ts

```diff
@@ -132,22 +135,30 @@ export class OrderService {
       searchObj.from = from;
       searchObj.size = _size;
     }
+    let count = null
+    let data = []
 
-    const total_orders = await this.elasticsearchService.count({
-      index: 'orders',
-      body: {
-        query: query,
-      },
-    });
-
-    const orders = await this.elasticsearchService.search(searchObj);
-
+    try {
+       const total_orders = await this.elasticsearchService.count({
+        index: 'orders',
+        body: {
+          query: query,
+        },
+      });
+  
+      const orders = await this.elasticsearchService.search(searchObj);
+      count = total_orders
+      data = orders.body.hits.hits
+    } catch (error) {
+      console.log('API "orders/list/:customerId":', error.body.error.reason);
```

Comment:
Instead of `console.log` use Loggers instead.

https://docs.nestjs.com/techniques/logger

Reply:
Either throw a new exception or return an `HttpException`. Don't return default values when error.